core/server/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -61,15 +61,24 @@
   /** The clock to use for determining the time. */
   protected final Clock mClock;
 
+  /** Keeps track of operation metrics. */
+  protected final MasterSource mMasterSource;
```

Comment:
It isn't strictly necessary, but since we're using it in ~30 places I think it's better to give it a name. Actually, I think it makes sense to get rid of `MasterContext` as a field here. Once the context reaches the constructor, we won't need to pass it anywhere else, so we can unpack it.